root/static/scripts/guess-case/utils.js

```diff
@@ -87,6 +87,14 @@ var lowerCaseBracketWordsList = [
 
 var lowerCaseBracketWords = new RegExp('^(' + lowerCaseBracketWordsList.join('|') + ')$', 'i');
 
+exports.turkishUpperCase = function(str) {
+  return str.replace(/i/g, 'İ').toUpperCase();
+};
+
+exports.turkishLowerCase = function(str) {
+  return str.replace(/I\u0307/g, 'i').replace(/I/g, 'ı').replace(/İ/g, 'i').toLowerCase();
```

Comment:
Which browser did you test that on? In Firefox, `"\u0130". toLowerCase() == "i" ` is true. (`"I\u307". toLowerCase() == "i"` is not, of course, but that’s what the first `replace()` is for.)